crates/core/src/operations/load.rs

```diff
@@ -22,6 +23,8 @@ pub struct LoadBuilder {
     log_store: LogStoreRef,
     /// A sub-selection of columns to be loaded
     columns: Option<Vec<String>>,
+    // Optional session options
+    session_config: Option<SessionConfig>
 }
```

Comment:
Here is where encryption is added for loading files, as a general session config param.